docs/http.md

```diff
@@ -481,6 +487,13 @@ step.
   direction of travel immediately after the maneuver.
 - `type` A string indicating the type of maneuver. **new identifiers might be introduced without API change**
    Types  unknown to the client should be handled like the `turn` type, the existance of correct `modifier` values is guranteed.
+- `lanes`: the available turn lanes at the turn
+  - `indication`: markings on the road, following the OSM scheme (e.g. left, slight_right, or through). Markings mey be combined into a list separated by ';'.
+  - `valid`: a boolean flag indicating whether the lane is a valid choice in the maneuver
+
+Currently, the supported lane tags are: `sharp_left, left, slight_left, sharp_right, right, slight_right, through, reverse, none`.
```

Comment:
We should make this consistent with our modifiers. We don't use underscore there. Suggestion:

| `direction`      | Description                               |
| ------------------- | ---------------------------------------|
| uturn             | lane used for uturn maneuver          |
| sharp right       | lane for a sharp right turn                        |
| right             | lane for  a normal turn to the right                |
| slight right      | lane for a slight turn to the right                |
| straight          | lane to go straight           |
| slight left       | lane for a slight turn to the left                 |
| left              | lane for a normal turn to the left                 |
| sharp left        | lane for a sharp turn to the left                  |
| all | lane allows turns in all direction |

We should also change `indication` -> `direction` or `type`.